.circleci/config.yml

```diff
@@ -96,6 +96,16 @@ workflows:
       - ruby-262-unittest
       - license_check
       - help_test
+
+  integration:
+    jobs:
+      - manual_approval: # <<< A job that will require manual approval in the CircleCI web application.
+          type: approval # <<< This key-value pair will set your workflow to a status of "On Hold"
+          # On approval of the `hold` job, any successive job that requires the `hold` job will run.
+      - integration_tests:
```

Comment:
What would be the idea of this during each PR?
If I have understood correctly what this does is that it requires the user interaction in CircleCI to approve the step. Would this be left as an optional job in GitHub? what would be the criteria?

Reply:
Integration tests are run nightly.

With this new `integration` workflow, each PR integration tests could be run, always manually approved. Useful if you suspect something could be broken